src/main/kotlin/net/atos/zac/admin/ZaakafhandelParameterBeheerService.kt

```diff
@@ -35,11 +34,11 @@ import java.util.UUID
 @AllOpen
 @NoArgConstructor
 @Suppress("TooManyFunctions")
-class ZaakafhandelParameterBeheerService @Inject constructor(
+open class ZaakafhandelParameterBeheerService @Inject constructor(
     private val entityManager: EntityManager,
     private val ztcClientService: ZtcClientService,
     private val zaakafhandelParameterService: ZaakafhandelParameterService,
-    private val smartDocumentsTemplatesService: SmartDocumentsTemplatesService
+//    private val smartDocumentsTemplatesService: SmartDocumentsTemplatesService
```

Comment:
I don't really like commented out code. It tends to cause confusion and stick around a long time if you are unlucky. I would just remove this code and save the work in some branch. :-)